community/modules/scheduler/SchedMD-slurm-on-gcp-controller/README.md

```diff
@@ -33,8 +33,7 @@ controller for optimal performance at different scales.
 This creates a controller node connected to the primary subnetwork with 1 login
 node (defined elsewhere). The controller will also have the `homefs` file system
 mounted via the `use` field and manage one partition, also declared in the `use`
-field. For more context see the
-[hpc-cluster-small example](../../../../examples/hpc-cluster-small.yaml).
+field.
```

Comment:
nit, you are not pointing to the legacy example?

Reply:
Will point to the `hpc-slurm-legacy` once it's added.